gradio/routes.py

```diff
@@ -283,6 +283,8 @@ async def join_queue(websocket: WebSocket):
                 return
             event.hash = e_hash["hash"]
             rank = Queue.push(event)
+            if rank is None:
+                await event.send_message({"msg": "queue_full"})
```

Comment:
let's disconnect the ws here? No need to continue running it IMO. Btw we will be handling ws disconnections in Frontend in the future(#2019), so it is ok if you want to always dc from frontend and handle in backend.

`await event.disconnect()
`we could even disconnect with a code if you can utilize that in frontend
`await event.disconnect(429)`

Reply:
Done